content/en/blog/falco-on-gke/index.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Adding runtime threat detection to Google Kubernetes Engine with Falco"
+linktitle: "Adding runtime threat detection to Google Kubernetes Engine with Falco"
+date: 2023-11-16
+author: Mike Coleman
+slug: falco-on-gke
+images:
+  - /blog/falco-on-gke/images/falco-on-gke-featured.png
+tags: ["Falco", "GKE", "Google Kubernetes Engine", "Google Cloud"]
+---
+
+
+One of the big advantages of running your workloads on a managed Kubernetes service like Google Kubernetes Engine (GKE) is that Google ensures your clusters are being deployed and managed following industry best practices. 
+
+While GKE clusters are incredibly secure and reliable, there is always room for improvement. 
+
+In this blog, we’re going to describe how you can enhance GKE’s already great security by adding runtime threat detection with Falco. 
+
+
+## What is Falco?
+
+Falco is a Cloud Native Computing Foundation project that provides runtime threat detection. Out of the box, Falco examines syscalls to alert you to any suspicious activity. And, since containers share the same kernel as their host, Falco can monitor not only activity on the host but also all the containers running on that host. Moreover, Falco pulls data from both Kubernetes and the container runtime to add additional context to its alerts. 
```

Comment:
```suggestion
Falco is a Cloud Native Computing Foundation project that provides runtime threat detection. Out of the box, Falco examines syscalls to alert you to any suspicious activity. And, since containers share the same kernel as their host, Falco can monitor not only activity on the host but also activity on all of the containers running on that host. Moreover, Falco pulls data from both Kubernetes and the container runtime to add additional context to its alerts. 
```